dist/setup-appliance.sh

```diff
@@ -427,16 +427,34 @@ function ask {
 
 }
 ###############################################################################
+function check_service {
+    srv=$1
+    STATE=$(chkconfig $srv 2>/dev/null|awk '{print $2}')
+    if [[ $STATE == "" ]]; then
+        ENABLED=`systemctl is-enabled $srv`
+        if [[ "$ENABLED" == "enabled" ]];then
+            STATE=on
+            ACTIVE=`systemctl is-active $srv`
+            [[ "$ACTIVE" == "active" ]] || systemctl start $srv
+        fi
+    fi
+    if [[ $STATE != on ]];then
+      ask "Service $srv is not enabled. Would you like to enable it? [Yn]" "y"
```

Comment:
The user should only be asked for RECOMMENDED_SERVICES or OPTIONAL_SERVICES, 
but not for REQUIRED_SERVICES